common/config.go

```diff
@@ -1,57 +1,72 @@
 package common
 
 import (
+	"flag"
 	"github.com/BurntSushi/toml"
 	"github.com/Sirupsen/logrus"
 	"github.com/caarlos0/env"
+	"os"
 	"path"
 )
 
 const configFileDir = "/etc/infoblox"
 
+type Config interface {
+	LoadFromConfFile() error
+	LoadConfig() error
+}
+
 type GridConfig struct {
```

Comment:
Thanks for reviewing Ko! Yes I will update the documentation in a separate patch. I have lots left to document :)